3]. Competitive Programming/LeetCode/Problems/_0051)_N-Queens.java

```diff
@@ -0,0 +1,74 @@
+// Link to Problem : https://leetcode.com/problems/n-queens/
+class NQueens_51 {
```

Comment:
change name `NQueens_51` to `Solution`
then code accepted

Reply:
Done. 👍